pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -2,40 +2,53 @@ package ovirt
 
 import (
 	ovirtsdk4 "github.com/ovirt/go-ovirt"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 
 	"github.com/openshift/installer/pkg/types/ovirt"
 )
 
+const PlatformValidationMaxTries = 3
+
 // Platform collects ovirt-specific configuration.
 func Platform() (*ovirt.Platform, error) {
 	p := ovirt.Platform{}
 
+	var c *ovirtsdk4.Connection
+
+	// Fetch config from file
 	ovirtConfig, err := NewConfig()
-	if err != nil {
-		ovirtConfig, err = engineSetup()
+	for tries := 0; tries < PlatformValidationMaxTries; tries++ {
+		if err == nil {
```

Comment:
You need to switch the order of the if's, meaning:
```
 	ovirtConfig, err := NewConfig()
	for tries := 0; tries < PlatformValidationMaxTries; tries++ {
		if err != nil {
                    ..........
                }
                if err == nil {
                    ..........
                }
```

That way you always get the same number of tries and validations